### DISCUSSION TITLE
Issue template by ".github" repository still exists after that repository removed

### DISCUSSION THREAD

**Discussion by gnuhead-chieb** [community/community#38367] [Projects and Issues]
## Steps to reproduce
1. Create repository named ".github"
2. Create issue template in that repository
3. See this issue template in another repository
4. Remove ".github" repository
5. Issue template still exist in another repository

## Temporary solution
Remove repository, create blank repository again, and then remove

**Comment by ossanna16** [comment]
Hi there @gnuhead-chieb and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our official introduction thread
To connect with other new users, check out this Discussion to Meet Your GitHub Community Team, further your GitHub knowledge by working through our GitHub Skill 🆙 exercises, and participate in our weekly Fun Friday chats
